cmd/axelard/cmd/vald/config/config.go

```diff
@@ -11,22 +11,26 @@ import (
 
 // ValdConfig contains all necessary vald configurations
 type ValdConfig struct {
-	tss.TssConfig   `mapstructure:",squash"`
-	BroadcastConfig `mapstructure:",squash"`
-	BatchSizeLimit  int `mapstructure:"max_batch_size"`
-	BatchThreshold  int `mapstructure:"batch_threshold"`
+	tss.TssConfig      `mapstructure:",squash"`
+	BroadcastConfig    `mapstructure:",squash"`
+	BatchSizeLimit     int           `mapstructure:"max_batch_size"`
+	BatchThreshold     int           `mapstructure:"batch_threshold"`
+	MaxOutOfSyncHeight int64         `mapstructure:"max_out_of_sync_height"`
+	MaxBlockTime       time.Duration `mapstructure:"max_block_time"`
```

Comment:
The max "block time", time to produce one block. Since we check the timestamp of the latest block, this param should be larger than the block times otherwise vald might conclude that the node is still out of sync. Added a comment.